hellomama_registration/settings.py

```diff
@@ -14,7 +14,7 @@
 import djcelery
 import dj_database_url
 import mimetypes
-
+from hellomama_registration.utils import MSG_TYPES
```

Comment:
I was worried about changing the functionality too much. This way it is defined statically in code, like it was previously, whereas putting it in settings makes it a dynamic thing that can be overridden in your setting file (which is possibly better), but I'm not sure what the reason was for making it static in the first place.

I don't see any reason to not put it in the settings file though